src/core/qgsvectorlayerutils.h

```diff
@@ -319,6 +319,16 @@ class CORE_EXPORT QgsVectorLayerUtils
      * \since QGIS 3.12
      */
     static QString getFeatureDisplayString( const QgsVectorLayer *layer, const QgsFeature &feature );
+
+    /**
+     * \returns true if at least one feature of the \a fids on \a layer is connected as parent in at
```

Comment:
```suggestion
     * \returns TRUE if at least one feature of the \a fids on \a layer is connected as parent in at
```

Will be replaced in api docs with the correct values for for Python/C++